resources/views/docs/_structures/beatmapset_compact.md

```diff
@@ -11,34 +11,38 @@ creator         | string                              | |
 favourite_count | number                              | |
 id              | number                              | |
 nsfw            | boolean                             | |
+offset          | number                              | |
 play_count      | number                              | |
 preview_url     | string                              | |
 source          | string                              | |
 status          | string                              | |
+spotlight       | boolean                             | |
 title           | string                              | |
 title_unicode   | string                              | |
 user_id         | number                              | |
 video           | boolean                             | |
 
 Those fields are optional.
 
-Field                   | Type                  | Description
------------------------ | --------------------- | -----------
-beatmaps                | [Beatmap](#beatmap)[] | |
-converts                |                       | |
-current_user_attributes |                       | |
-description             |                       | |
-discussions             |                       | |
-events                  |                       | |
-genre                   |                       | |
-has_favourited          | boolean               | |
-language                |                       | |
-nominations             |                       | |
-pack_tags               | string[]              | |
-ratings                 |                       | |
-recent_favourites       |                       | |
-related_users           |                       | |
-user                    |                       | |
+Field                   | Type                         | Description
+----------------------- | ---------------------------- | -----------
+beatmaps                | [Beatmap](#beatmap)[]        | |
+converts                |                              | |
+current_nominations     | [Nomination](#nomination)[]? | |
```

Comment:
this field isn't nullable

Reply:
ah, I guess you guys make a distinction between "nullable" and "not included at all". Fair enough.

Reply:
yeah. I think there are some optional fields that's nullable.